### ISSUE TITLE
Summary: Exceeds Expectations

### ISSUE THREAD

**Issue by Caealana** [drnanjo/intro-data-capstone-biodiversity#7]
I had a lot of fun going over your project. I was very impressed with the amount of detail and work you put into this project! You made many extra visualizations and tables to thoroughly explore the data. I especially liked your pie chart of the locations for the sheep. It was very clear and easy to interpet. All of the data visualizations that ended up on your slides are of very high quality. One minor issue:
<img width="398" alt="screenshot at mar 06 13-08-16" src="https://user-images.githubusercontent.com/14001120/37046760-77382c0c-213f-11e8-83e8-bf4dbff3ab25.png">
The legend you have doesn't quite make sense. Every bar is blue, and the audience won't know what you mean by blue meaning scientific_name. 

You followed many of the best conventions concerning indentation and variable naming conventions. This helped in making your python code easily readable. [Here](https://www.python.org/dev/peps/pep-0008/) is more information on python styling conventions if you are interested.

 I would suggest you practice adding in your own comments and a README to this github repository to give some context to your code. Comments answer questions like 'What were you thinking when you writing this code?' and 'What is the purpose of this chunk of code? Are there currently any issues with it?" [Here ](https://guides.github.com/features/wikis/) is a helpful guide for creating READMEs that give a good introduction to your project.

Your presentation had great flow and story building. It was very easy to follow your thought process. You did not let any slide get too wordy or cluttered. I would suggest experimenting with highlighting and bolding on some of your slides.
For example:
<img width="746" alt="screenshot at mar 06 13-14-13" src="https://user-images.githubusercontent.com/14001120/37047087-5b345fa2-2140-11e8-9551-ac65132fb6cc.png">
For this slide, you can bold the conservation statuses (species of concern, threatened, etc.) to indicate they are the main takeaways from this slide.

**Comment by drnanjo** [comment]
Thank you. I am employed as a college professor, did most of the training over my break and had to push myself to finish the project once the term began. I need to know if I have to do the fixes you recommend now, or I can do them for my own enrichment when I have my next break.

I enjoy working with Jupyter Notebooks, and I enjoyed the tutorial immensely. I know I need more practice. I work very hard to make time. It's tough. Thank you for your kind feedback.

Let me know if I need to do anything else.
